sal_interfaces/Dome/Dome_Commands.xml

```diff
@@ -1,241 +1,133 @@
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Explanation>http://sal.lsst.org</Explanation>
-    <item>
-        <EFDB_Name>temperature</EFDB_Name>
-        <Description>Set the temperature to maintain in the dome heat sources</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>deg_C</Units>
-        <Count>1</Count>
-    </item>
-</SALCommand>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_crawlAz</EFDB_Topic>
+        <Description>Move the azimuth axis at constant velocity.</Description>
+        <item>
+            <EFDB_Name>velocity</EFDB_Name>
+            <Description>The velocity. Positive is the direction of increasing azimuth.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg/second</Units>
+            <Count>1</Count>
+        </item>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_crawlEl</EFDB_Topic>
+        <Description>Move the elevation axis (light/wind screen) at constant velocity.</Description>
+        <item>
+            <EFDB_Name>velocity</EFDB_Name>
+            <Description>The velocity. Positive is the direction of increasing azimuth.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg/second</Units>
+            <Count>1</Count>
+        </item>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_moveAz</EFDB_Topic>
+        <Description>Move the azimuth axis at constant velocity, starting from a specified position.</Description>
+        <item>
+            <EFDB_Name>azimuth</EFDB_Name>
+            <Description>The position.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>velocity</EFDB_Name>
+            <Description>The velocity. Positive is the direction of increasing azimuth.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg/second</Units>
+            <Count>1</Count>
+        </item>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_moveEl</EFDB_Topic>
+        <Description>Move the elevation axis (light/wind screen) at constant velocity, starting from a specified position.</Description>
+        <item>
+            <EFDB_Name>elevation</EFDB_Name>
+            <Description>The position.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg</Units>
+            <Count>1</Count>
+        </item>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_park</EFDB_Topic>

```

Comment:
Sorry to be picky because I really appreciate that you're making all these changes but a Description would be welcome here.

Reply:
I agree, but I'd have to know what it does first! I am guessing it moves the azimuth to a given "park" position, but what about elevation, the shutter, etc.?

Reply:
That is a good point which we haven't discussed yet. For now the mock controller assumes that only the dome azimuth rotation gets parked and the rest not.